docs/release_notes.rst

```diff
@@ -1,34 +1,22 @@
 Release Notes
 =============
 
-v3.2.0rc2
```

Comment:
these still exist... i think we need to leave them in

Reply:
I think we need to leave the tags -- I think it's fine to combine the release notes though since all the "release candidates" are still version `3.2.0`